enterprise/views.py

```diff
@@ -185,18 +126,31 @@ def get_course_specific_consent(self, request, course_id):
         platform_name = configuration_helpers.get_value("PLATFORM_NAME", settings.PLATFORM_NAME)
         course_name = course_details['name']
         data = {
+            'consent_request_prompt': _(
```

Comment:
So we have some strings embedded directly in the template and some strings specified here in the context data -- I'd prefer all strings to be provided to the template via the context, since it gives us a lot more control over what to display.